c31-swynlake/reactive-nutshell
# Bm message visibility

# Description
Adding Chat Room to /messages path. Chat room should show maximum of 10 of the most recent messages. For each message from the user there should be an edit button on that message. An input field is also available to input and save a new message. The messages should update as new messages are posted.

Applies to issue # (11, 12, 13)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1. `git fetch --all`
2. `git checkout bm-message-visibilty`
3. Login using userName and password of existing user in your database. Click on messages tab in navbar. Verify that no more than 10 messages are visible. Verify that any of your current user messages have an edit button on them. Input and save a new message, verify that it posts to database and is visible in the chat room. Verify that if you click edit button an input field opens for you to edit, and that when you click save the edited message is saved to database.

# Checklist:
